fenix/app/src/androidTest/java/org/mozilla/fenix/ui/SearchTest.kt

```diff
@@ -128,29 +156,41 @@ class SearchTest {
         }
     }
 
+    // Verifies temporarily changing the search engine from the Search Shortcut menu
```

Comment:
nit: // Verifies a temporary change of search engine form the Search shortcut menu